lua/zenbones/shipwright/runners/windows_terminal.lua

```diff
@@ -0,0 +1,40 @@
+local template = [[{
+"name": "zenbonse",
+"foreground": "$fg",
+"background": "$bg",
+"selectionBackground": "$selection_bg",
+"cursorColor": "$cursor_bg",
+"black": "$black",
+"red": "$red",
+"green": "$green",
+"yellow": "$yellow",
+"blue": "$blue",
+"purple": "$magenta",
+"cyan": "$cyan",
+"white": "$white",
+"brightBlack": "$bright_black",
+"brightRed": "$bright_red",
+"brightGreen": "$bright_green",
+"brightYellow": "$bright_yellow",
+"brightBlue": "$bright_blue",
+"brightPurple": "$bright_magenta",
+"brightCyan": "$bright_cyan",
+"brightWhite": "$bright_white"
+}
+]]
+
+local helpers = require "shipwright.transform.helpers"
+
+---@diagnostic disable: undefined-global
+-- selene: allow(undefined_variable)
+run(
+	{ specs, p, term },
+	transform.colorscheme_to_term_colors,
```

Comment:
Like this:

```suggestion
	transform.colorscheme_to_term_colors,
	function(colors)
		return vim.tbl_extend("keep", colors, { name = name })
	end
```

See: https://github.com/mcchrish/zenbones.nvim/blob/main/lua/zenbones/shipwright/runners/kitty.lua#L5-L19

Reply:
😅